src/officehours_api/migrations/0009_auto_20200325_1220.py

```diff
@@ -14,7 +15,7 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name='meeting',
             name='backend_metadata',
-            field=jsonfield.fields.JSONField(null=True),
+            field=models.JSONField(null=True),
```

Comment:
While this doesn't seem advisable, I did find a suggestion in the Django forms that editing the migration file would be an appropriate solution when migrating from a third-party implementation of a JSON field to the native `JSONField` released in Django 3.1: https://forum.djangoproject.com/t/deprecation-warning-in-unmaintained-3rd-party-app-used-in-old-migrations-django-jsonfield/10819

I tested locally and Django doesn't complain about unapplied changes after editng `models.py` to use the new one. And preexisting data loads correctly.